client/components/domains/domain-registration-suggestion/index.jsx

```diff
@@ -195,22 +162,84 @@ class DomainRegistrationSuggestion extends React.Component {
 						} )
 					: translate( 'Select', { context: 'Domain mapping suggestion button' } );
 		}
+		return {
+			buttonClasses,
+			buttonContent,
+		};
+	}
+
+	getPriceRule() {
+		const { cart, domainsWithPlansOnly, selectedSite, suggestion } = this.props;
+		return getDomainPriceRule( domainsWithPlansOnly, selectedSite, cart, suggestion );
+	}
+
+	renderDomain() {
+		const { suggestion: { domain_name: domain }, isFeatured } = this.props;
+		return (
+			<h3 className="domain-registration-suggestion__title">
+				{ domain }
+				{ ! isFeatured ? this.getDomainFlags() : null }
+			</h3>
+		);
+	}
+
+	renderProgressBar() {
+		const { suggestion: { isRecommended, isBestAlternative }, translate, isFeatured } = this.props;
+
+		if ( ! isFeatured ) {
+			return null;
+		}
+
+		let title, props;
```

Comment:
`props` is a bit too close too `this.props` - I can totally someone else coming here and noticing this "mistake" and fixing `props` to `this.props`. So I'd call it something more explicit like `progressBarProps`.

Reply:
Good point, I like `progressBarProps`.